config.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"net"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+type Config struct {
+	AWSRegion     string `envconfig:"AWS_REGION" required:"true"`
+	DogstatsdHost string `split_words:"true" required:"true"`
+	DogstatsdPort string `split_words:"true" default:"8125"`
+	QueueName     string `split_words:"true" required:"true"`
+	ServicePort   string `split_words:"true" required:"true"`
+}
+
+func (e *Config) DataDogAddress() string {
+	return net.JoinHostPort(e.DogstatsdHost, e.DogstatsdPort)
+}
+
+func FromEnv() (*Config, error) {
+	var env *Config
```

Comment:
I thought it needs to be a pointer because the function returns a pointer and in the case of an error I can return a `nil` pointer for the config. I did remove the `&` from the `envconfig.Process()` call while leaving the `*Config` type on `env`.

Reply:
Either the `var env *Config` pointer type, or the `&env` indirection in the `Process` call was redundant. When we had both then `&env` is a `**Config`. Changing either works 👍